manifests/install.pp

```diff
@@ -247,14 +248,22 @@
         'RedHat': {
           if $pip_ensure != 'absent' {
             if $python::use_epel == true {
-              include 'epel'
-              if $python::manage_pip_package { Class['epel'] -> Package['pip'] }
-              if $python::manage_python_package { Class['epel'] -> Package['python'] }
+              include epel
+
+              if $python::manage_pip_package {
+                Class['epel'] -> Package['pip']
```

Comment:
we could instead just `require epel`, or? cc @alexjfisher

Reply:
yes

Reply:
Fixed.